# Hold information about timing.
Header header

# Name of the publisher node
string name

# EgoNoiseLoad Array
ego_noise_monitor/EgoNoiseLoad[] data